GUIDE.rst

```diff
@@ -79,21 +79,30 @@ This defines a service named ``KeyValue`` with two functions:
 ``setValue``
     a void function that takes in two parameters.
 
-Once you have defined your service, generate corresponding Thrift types by
-running the following:
+
+------------
+Thrift Types
+------------
+
+The Apache Thrift compiler is buggy and inconsistent across languages, so we’re
+going to dynamically generate our Thrift types.
+
+Open up ``keyvalue/thrift.py``:
 
 .. code-block:: bash
 
-    $ thrift --gen py:new_style,dynamic,slots,utf8strings \
-        -out keyvalue thrift/service.thrift
+    $ cat > keyvalue/thrift.py
+    from tchannel import thrift
 
-This generates client- and server-side code to interact with your service.
+    service = thrift.load(path='thrift/keyvalue.thrift', service='keyvalue')
```

Comment:
Do we want to recommend the approach of replacing the current module/installing a submodule?